etc/config/c3.amazon.properties

```
@@ -0,0 +1,8 @@
+compilers=c3c
+compilerType=c3c
+defaultCompiler=c3c
+supportsBinary=false
+supportsBinaryObject=false
+supportsExecute=false
+compiler.c3c.name=c3c 0.4
+compiler.c3c.exe=/opt/compiler-explorer/c3-0.4/c3c
```

This creates a group which sets the base name and marks the compilers as semver. It will make it so compilers show up with the name `$baseName $semver`

```suggestion
compilers=&c3c
compilerType=c3c
defaultCompiler=c3c04
supportsBinary=false
supportsBinaryObject=false
supportsExecute=false
group.c3c.compilers=c3c04
group.c3c.isSemVer=true
group.c3c.baseName=c3
compiler.c3c04.semver=0.4
compiler.c3c04.exe=/opt/compiler-explorer/c3-0.4/c3c
```